src/description/eth_getBlockTransactionCountByNumber.md

```diff
@@ -0,0 +1 @@
+* If the client has a block equal to the `block_number` parameter it **MUST** return the number of transactions the block conatins, otherwise it **MUST** return null.
```

Comment:
Maybe this is what clients currently do, but it is *definitely* not what clients **should** do.

Clients **should** return a JSON-RPC error if they do not have the block requested.  If they do have the requested block, it should always return a number (never null).

In general, the JSON-RPC should be returning errors for error conditions, not returning sentinel values or null for error conditions.